### PR TITLE
Add release script placeholder files.

### PR DESCRIPTION
**Pull request by jlinder**
We are automating the release process. These files are placeholders for
the new processes steps.

Release note: None

### CODE REVIEW

File: build/release/teamcity-compile-and-publish-prerelease-build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (otan) [inline build/release/teamcity-compile-and-publish-prerelease-build.sh:1]: make sure the permissions for these are `755`

**Comment by jlinder** [comment]
I opened this as draft since there ones for MarkAsInternallyTested and MarkAsProductionTested aren't certain yet. (arguably, we could get these ones in and then add those once we know their names).

File: build/release/teamcity-compile-and-publish-prerelease-build.sh (hunk 1 above)
Comment (jlinder) [reply build/release/teamcity-compile-and-publish-prerelease-build.sh:1]: The are now `755`. Originally they were `744`.

**Comment by jlinder** [comment]
Let's just get these in.

**Review by otan — APPROVED** [review]

**Comment by jlinder** [comment]
TFTR!

bors r=otan

**Pull request merged**